docs/man_pages/project/testing/test-visionos.md

```
@@ -0,0 +1,46 @@
+* Verify that [you have configured your project for unit testing](test-init.html) with the Vitest framework.
+* Verify that [you have stored your unit tests in `app` &#8594; `tests`](http://docs.nativescript.org/testing).
+* Verify that [you have configured your system and devices properly](http://docs.nativescript.org/testing).
+
+### Related Commands
+
+Command | Description
+--------|------------
+[test init](test-init.html) | Configures your project for unit testing with a selected framework.
+[test android](test-android.html) | Runs the tests in your project on Android devices or native emulators.
+[test ios](test-ios.html) | Runs the tests in your project on iOS devices or the iOS Simulator.
+<% } %>
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)